config/leviton/vrcs2.xml

```diff
@@ -1,21 +1,45 @@
-    <MetaDataItem id="0206" name="FrequencyName" type="0201">U.S. / Canada / Mexico</MetaDataItem>
-    <MetaDataItem name="Description">The Vizia RF + 4-Button Zone Controller with Switch (VRCZ4-MR) and 4-Button Scene Controller with Switch (VRCS4-MR) are perfect for retrofit applications because they replace an existing switch giving the user control of the local load as well as the benefit of scene or zone control.
- Item Description
-Vizia RF + 4-Button Zone Controller with Switch for Multi-Location Control with IR Remote Capability. White face assembled on device, ivory and light almond faces included.</MetaDataItem>
-    <MetaDataItem name="ProductPage">http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=VRCZ4-MRZ&amp;section=44147&amp;minisite=10251</MetaDataItem>
-    <MetaDataItem id="0206" name="Identifier" type="0201">VRCZ4-MR</MetaDataItem>
-    <MetaDataItem name="ProductSupport">http://www.leviton.com/OA_HTML/Home.jsp?minisite=10251&amp;respid=22372</MetaDataItem>
-    <MetaDataItem name="Name">VRCZ4-MR</MetaDataItem>
+    <MetaDataItem id="0243" name="ZWProductPage" type="1102">https://products.z-wavealliance.org/products/286/</MetaDataItem>
+    <MetaDataItem id="0243" name="FrequencyName" type="1102">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0243" name="Identifier" type="1102">VRCS2-MRZ</MetaDataItem>
     <ChangeLog>
       <Entry author="Justin Hammond - Justin@dynam.ac" date="03 May 2019" revision="1">Initial Metadata Import from Z-Wave Alliance Database - https://products.z-wavealliance.org/products/286/xml</Entry>
       <Entry author="Justin Hammond - Justin@dynam.ac" date="03 May 2019" revision="2">Updated Metadata Import from Z-Wave Alliance Database - https://products.z-wavealliance.org/products/287/xml</Entry>
       <Entry author="Justin Hammond - Justin@dynam.ac" date="03 May 2019" revision="3">Updated Metadata Import from Z-Wave Alliance Database - https://products.z-wavealliance.org/products/288/xml</Entry>
+      <Entry author="Jeff Brown - jeff.brown@gmail.com" date="29 June 2020" revision="4">Fixed metadata, added groups and indicators</Entry>
     </ChangeLog>
   </MetaData>
-  <CommandClass id="112"/>
+
+  <!-- Association Groups -->
+  <CommandClass id="133">
+    <Associations num_groups="2">
+      <!-- Each button toggles an individual scene on or off.  Multiple scenes can be active at once.
+           By default the device will associate groups 1 and 2 with the local loads.
+           Do not auto associate the controller with group 1. -->
+      <Group index="1" max_associations="232" label="Button 1" auto="false"/>
+      <Group index="2" max_associations="232" label="Button 2"/>
+    </Associations>
+  </CommandClass>
+
+  <!-- Indicator Lights -->
+  <!-- Use these messages to override the indicator lights.
+
+       91 00 1D 0D 01 00 00 : Reset LEDs to locally controlled operation (default behavior).
```

Comment:
Assuming I'm reading this correctly - its using the Manufactuter_Proprietary CC for Leds?

Reply:
That's correct.  Unfortunately it doesn't support the "Indicator" command class.